public/wp-content/themes/ColoredCow/src/scss/style.scss

```diff
@@ -34,4 +34,40 @@ Text Domain: coloredcow
 	}
 }
 
+.btn:hover{
+	cursor: pointer;
+}
+
+.images_carousel{
+	position: center;
+	display: block;
+	width:765px;
+	height:500px;
+}
+
+.about{
+	padding-left: 22px;
+	color:$white;
+	text-align: justify;
+}
+
+.eventdata{
+	color: $white;
+	text-align: center;
+}
+
+body{
+	background-image: url('dist/img/bg.png');
+	background-repeat: no-repeat;
+}
+
+.footer {
+	text-align:center;
+	bottom: 0;
+	width: 100%;
+	height:60px;
+    background-color:$black;
```

Comment:
I don't know why github is showing improper indentation, but it is properly indented in the editor